recovery-contracts/RecoveryWallet.sol

```diff
@@ -0,0 +1,65 @@
+pragma solidity 0.8.10;
+import "./RecoveryForwarder.sol";
+import "./CloneFactory.sol";
+/**
+ *
+ * RecoveryWallet
+ * ============
+ *
+ * Basic singleSig wallet designed to recover funds.
+ *
+ */
+contract RecoveryWallet is CloneFactory {
+
+  // Public fields
+  address public immutable signer;
+  address public immutable forwarderImplementationAddress;
```

Comment:
Added public modifier so that we can  view the value of `signer` and `forwarderImplementationAddress` and as we are initializing the variables in constructor added `immutable`